Tests/Engine/CustomizedRule.tests.ps1

```diff
@@ -38,40 +42,40 @@ Describe "Test importing customized rules with null return results" {
 
 Describe "Test importing correct customized rules" {
 
-	if(-not $IsCoreCLR)
-	{
-		Context "Test Get-Help functionality in ScriptRule parsing logic" {
-			It "ScriptRule help section must be correctly processed when Get-Help is called for the first time" {
-
-				# Force Get-Help to prompt for interactive input to download help using Update-Help
-				# By removing this registry key we force to turn on Get-Help interactivity logic during ScriptRule parsing
-				$null,"Wow6432Node" | ForEach-Object {
-					try
-					{
-						Remove-ItemProperty -Name "DisablePromptToUpdateHelp" -Path "HKLM:\SOFTWARE\$($_)\Microsoft\PowerShell" -ErrorAction Stop
-					} catch {
-						#Ignore for cases when tests are running in non-elevated more or registry key does not exist or not accessible
-					}
+	# if(-not ([bool] $IsCoreCLR))
+	# {
```

Comment:
This looks like it might need to be removed

Reply:
Fixed in b5407379293aa0c3d9b2451a979c82180dfb1e73